client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClientConfigImpl.java

```diff
@@ -93,40 +121,62 @@ public Builder self()
         }
 
         /**
-         * Sets the {@code maxRetries} and returns a reference to this Builder enabling method chaining.
+         * Sets the {@code maxRetries} and returns a reference to this Builder enabling method chaining
          *
          * @param maxRetries the {@code maxRetries} to set
          * @return a reference to this Builder
          */
         public Builder maxRetries(int maxRetries)
         {
-            return update(b -> b.maxRetries = maxRetries);
+            return update(builder -> builder.maxRetries = maxRetries);
```

Comment:
unnecessary (and unrelated) change

Reply:
Reverted to single-letter names here and in several other places.